### PR TITLE
Fixing some links

### LINKED ISSUE #961: Fixing some links
**Description**

Fix some links to work with the new community name from `Jaoude` to `Hub`

### PR DESCRIPTION
**Pull request by TawfikYasser**
closes #961

#### What does this PR do?

Replace old community link to new

### CODE REVIEW

**Review by Nitesh-thapliyal — DISMISSED** [review]

File: Finding-Open-Source-Projects.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ So, you decided you want to start contributing to open source. Great :thumbsup:!
 We know this can be a difficult process, especially if it's your first time contributing to open source. For that reason, we have assimilated a list of tips to help you find a project to contribute to. The list also contains links to other resources about this topic from various authors, to help you further in finding projects that welcome contributors such as yourself :smiley:.
 
 ## Tips
-- Look at our projects in the [Eddie Jaoude Community](https://github.com/EddieJaoudeCommunity) GitHub organization.
+- Look at our projects in the [Eddie Hub Community](https://github.com/EddieHubCommunity) GitHub organization.
```
Comment (SashoStoichkov) [inline Finding-Open-Source-Projects.md:7]: `[EddieHub](https://github.com/EddieHubCommunity)`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](https://discord.gg/jZQs6Wu)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an issue to join the EddieJaoudeCommunity GitHub community.
+Raise an issue to join the EddieHubCommunity GitHub community.
```
Comment (SashoStoichkov) [inline README.md:5]: `Raise an issue to join the EddieHub GitHub community.`

File: README.md [hunk 3]
```diff
@@ -41,11 +41,11 @@ Please check out the [issue queue](https://github.com/EddieJaoudeCommunity/suppo
 	 
 	 - Showcase of Open Source Portfolios to get feedback from community members.
     
-      Feel free to check out other cool repositories of EddieJoaude Community 
-      <a href='https://github.com/EddieJaoudeCommunity'>here</a>.
+      Feel free to check out other cool repositories of EddieHub Community
```
Comment (SashoStoichkov) [inline README.md:44]: `EddieHub` ~~Community~~

File: README.md [hunk 4]
```diff
@@ -41,11 +41,11 @@ Please check out the [issue queue](https://github.com/EddieJaoudeCommunity/suppo
...
 	 - Showcase of Open Source Portfolios to get feedback from community members.
     
-      Feel free to check out other cool repositories of EddieJoaude Community 
-      <a href='https://github.com/EddieJaoudeCommunity'>here</a>.
+      Feel free to check out other cool repositories of EddieHub Community 
+      <a href='https://github.com/EddieHubCommunity'>here</a>.
 
 ## FAQ
-- **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [Eddie Jaoude Community](https://github.com/EddieHubCommunity).
+- **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [Eddie Hub Community](https://github.com/EddieHubCommunity).
```
Comment (SashoStoichkov) [inline README.md:48]: `[EddieHub](https://github.com/EddieHubCommunity)`

**Review by SashoStoichkov — CHANGES_REQUESTED** [review]
@TawfikYasser, thank you for the PR 🎉 

The name of the community is just `EddieHub` 🤓 
Here I suggest some changes ⬇️

**Comment by TawfikYasser** [comment]
I think now required changes are done. @SashoStoichkov

**Comment by SashoStoichkov** [comment]
LGTM 👍🏻 Thank you for the changes 💯

**Review by SashoStoichkov — APPROVED** [review]
LGTM 👍🏻 Thank you for the changes 💯

**Review by galahad42 — APPROVED** [review]
LGTM!

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:

**Comment by AllanRegush** [comment]
Good Work Everyone :tada:

**Comment by eddiejaoude** [comment]
Great team work everyone 👍 thank you 🤓

**Pull request merged**